wgengine/router/ifconfig_windows.go

```diff
@@ -408,15 +400,13 @@ func configureInterface(cfg *Config, tun *tun.NativeTun) (retErr error) {
 		return fmt.Errorf("syncAddresses: %w", err)
 	}
 
-	sort.Slice(routes, func(i, j int) bool { return routeLess(&routes[i], &routes[j]) })
+	sort.Slice(routes, func(i, j int) bool { return routeDataCompare(&routes[i], &routes[j]) == -1 })
```

Comment:
you can also do `slices.Sort(routes, routeDataLess)`

Reply:
done + tested